### PR TITLE
Display List properly + Status bar colour + Fix Patient so duplicate NRIC cannot be added

### PR DESCRIPTION
**Pull request by daytona65**
Need to figure out the css for status bar...
Otherwise list displays as expected
Fixed patient so only equivalent NRIC triggers the duplicate exception

### CODE REVIEW

File: src/main/resources/view/HelpWindow.css
Comment (KSunil2001) [inline src/main/resources/view/HelpWindow.css:1]: According to code style, is it okay to add single line comments the way you did? 🤔

File: src/main/resources/view/PatientListCard.fxml [hunk 1]
```diff
@@ -25,9 +24,12 @@
             <Region fx:constant="USE_PREF_SIZE" />
           </minWidth>
         </Label>
-        <Label fx:id="name" text="\$first" styleClass="cell_big_label" />
+        <Label fx:id="name" text="" styleClass="cell_big_label" />
+        <Label fx:id="status" styleClass="status-bar-with-border"/>
```
Comment (KSunil2001) [inline src/main/resources/view/PatientListCard.fxml:28]: Does status not require a text field?

File: src/main/resources/view/MedInfoTheme.css [hunk 2]
```diff
@@ -0,0 +1,352 @@
...
+    -fx-opacity: 1;
+}
+
+.label-header {
+    -fx-font-size: 32pt;
+    -fx-font-family: "Segoe UI Light";
+    -fx-text-fill: white;
+    -fx-opacity: 1;
+}
+
```
Comment (yitong241) [inline src/main/resources/view/MedInfoTheme.css:26]: decent color I like it

File: src/main/resources/view/PatientListCard.fxml (hunk 1 above)
Comment (yitong241) [reply src/main/resources/view/PatientListCard.fxml:28]: same question

**Review by yitong241 — CHANGES_REQUESTED** [review]
Should a field be added for status?

File: src/main/resources/view/PatientListCard.fxml (hunk 1 above)
Comment (daytona65) [reply src/main/resources/view/PatientListCard.fxml]: No need because I set it in the java file

Comment (daytona65) [reply src/main/resources/view/PatientListCard.fxml]: By default it is an empty text field anyway

File: src/main/resources/view/HelpWindow.css
Comment (daytona65) [reply src/main/resources/view/HelpWindow.css:1]: I followed what the original authors did for Extensions.css, they did single line comments there too
No Idea if that's right

**Comment by daytona65** [comment]
> Should a field be added for status?

What do you mean

File: src/main/java/seedu/address/ui/PatientCard.java [hunk 3]
```diff
@@ -41,6 +50,13 @@ public PatientCard(Patient patient, int displayedIndex) {
         this.patient = patient;
         id.setText(displayedIndex + ". ");
         name.setText(patient.getName().fullName);
+        nric.setText(patient.getNric().value);
+        String statusString = patient.getStatus().getDesc();
```
Comment (nramapurath) [inline src/main/java/seedu/address/ui/PatientCard.java:54]: Isn't this a violation of law of demeter? Maybe you can consider adding a getStatusDescription method in patient?

File: src/main/resources/view/Extensions.css [hunk 4]
```diff
@@ -1,10 +1,9 @@
 
 .error {
-    -fx-text-fill: #d06651 !important; /* The error class should always override the default text-fill style */
+    -fx-text-fill: #d06651 !important;
```
Comment (nramapurath) [inline src/main/resources/view/Extensions.css:3]: Just curious, why did you choose to remove the rationale in the comments?

File: src/main/java/seedu/address/ui/PatientCard.java (hunk 3 above)
Comment (daytona65) [reply src/main/java/seedu/address/ui/PatientCard.java:54]: Alright sounds good, will do that

**Pull request merged**